### PR TITLE
1665 rpb

### PR DESCRIPTION
**Pull request by TobiasNx**
I added a lookup for `rpbId`. Using the info from: https://github.com/hbz/lookup-tables/pulls

### CODE REVIEW

**Review by dr0i — APPROVED** [review]

File: src/main/resources/alma/maps/almaMmsId2rpbId.tsv
[the remark this answers is not in the document]
Comment (dr0i) [reply src/main/resources/alma/maps/almaMmsId2rpbId.tsv:1]: against lookup. I will fetch the data from there on a regular data then.
>  should not be deployed to production

Is that so? This is gonna be deployed early Monday. Shall I revert?

[the remark this answers is not in the document]
Comment (fsteeg) [reply src/main/resources/alma/maps/almaMmsId2rpbId.tsv:1]: I can update it to the version that should be deployed to production. Should I do the PR here or in lookup-tables?

[the remark this answers is not in the document]
Comment (dr0i) [reply src/main/resources/alma/maps/almaMmsId2rpbId.tsv:1]: Lookup-tables. I will fetch it then.

[the remark this answers is not in the document]
Comment (fsteeg) [reply src/main/resources/alma/maps/almaMmsId2rpbId.tsv:1]: To time it with the editing break by our RPB colleagues, I would update the file on Thursday.

**Pull request merged**